passman.py

```diff
@@ -77,7 +78,18 @@ def btn_Edit():
         renderCreateView(index=ind)
 
     def btn_CopyPassword():
-        print "copying password"
+        current_selection = entries_list.curselection() 
+        if current_selection:
+            selected_entry = entries_list.get(current_selection[0])
+            if selected_entry not in "No entries found!":
+                root.clipboard_clear()
+                root.clipboard_append(selected_entry)
+                tkMessageBox.showinfo("Copy Password", "Password has been copied to clipboard")
```

Comment:
I like the idea of giving the user feedback so they know the password was copied successfully, but I think popping up a dialog box that the user has to then click to close is a bit heavy-handed. Perhaps just having a label somewhere on the UI for the feedback would be a better approach. We want to try to keep things as simple as possible and minimize the number of clicks / user actions needed wherever possible.